.github/workflows/package-manager-demo.yml

```diff
@@ -72,3 +75,26 @@ jobs:
           PACKAGEMANAGER_TOKEN: ${{ secrets[matrix.package-manager.token] }}
         run: "./rspm add binary --source=local-api --distribution=${{ matrix.os.distro }} --path=$BINARY_FILE || echo 'Already uploaded'"
         shell: bash
+
+      - name: Setup Python 3
```

Comment:
You might consider making these two separate Github actions for folks that mostly want to copy & paste